cmd/fyne/internal/mobile/env.go

```diff
@@ -326,6 +326,13 @@ func archNDK() string {
 		arch = "x86"
 	case "amd64":
 		arch = "x86_64"
+	case "arm64":
+		// For darwin/arm64, see https://golang.org/cl/346153.
+		if runtime.GOOS == "darwin" {
```

Comment:
Why if darwin? I thought this was a fix for Android NDK not supporting arm64?

Reply:
It's for building android on a darwin/arm64 machine. Hence, GOOS == darwin, and the GOARCH is arm64.

I personally don't want to write the darwin check, but in the Go's CL review process, the Go team suggests to better just check for darwin, because on arm64 platforms, only darwin is (partially )supported in NDK. They don't want to risk an undefined behavior such as someone using linux/arm64 or windows/arm64 where NDK does not yet fully supported.

Besides, on a darwin/arm64 machine, using x86_64 is a temporary solution (yet) because NDK does not have any arm64 distribution at the moment.